src/components/MapView/AlertForm/index.tsx

```diff
@@ -0,0 +1,330 @@
+import React, { useMemo, useState } from 'react';
+import {
+  Box,
+  Button,
+  createStyles,
+  FormControl,
+  ListSubheader,
+  MenuItem,
+  Select,
+  TextField,
+  Theme,
+  Typography,
+  withStyles,
+  WithStyles,
+} from '@material-ui/core';
+import { faBell, faCaretDown } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { useSelector } from 'react-redux';
+import { getBoundaryLayerSingleton } from '../../../config/utils';
+import { BoundaryLayerProps, LayerKey } from '../../../config/types';
+import { LayerData } from '../../../context/layers/layer-data';
+import { layerDataSelector } from '../../../context/mapStateSlice/selectors';
+import LayerDropdown from '../Layers/LayerDropdown';
+
+// Not fully RFC-compliant, but should filter out obviously-invalid emails.
+const EMAIL_REGEX: RegExp = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+
+// This should probably be determined on a case-by-case basis,
+// depending on if the downstream API has the capability.
+// For now it can be permanently enabled.
+const ALERT_FORM_ENABLED = true;
+
+enum Statistic {
+  Maximum,
+  Minimum,
+  Median,
+  Mean,
+}
+
+enum Comparator {
+  '>',
+  '>=',
+  '<',
+  '<=',
+}
+
+const boundaryLayer = getBoundaryLayerSingleton();
+
+function AlertForm({ classes }: AlertFormProps) {
+  const boundaryLayerData = useSelector(layerDataSelector(boundaryLayer.id)) as
+    | LayerData<BoundaryLayerProps>
+    | undefined;
+
+  const [isAlertFormFormOpen, setIsAlertFormFormOpen] = useState(false);
+
+  // form elements
+  const [hazardLayerId, setHazardLayerId] = useState<LayerKey>();
+  const [regionsList, setRegionsList] = useState<string[]>(['allRegions']);
+  const [emailValid, setEmailValid] = useState<boolean>(false);
+  const [email, setEmail] = useState('');
+  const [selectedStat, setSelectedStat] = useState<Statistic>();
+  const [selectedComparator, setSelectedComparator] = useState('');
```

Comment:
after changing the enum, this should be `useState<Comparator>()` and be used similar to how to setup `selectedStat`